docs/operations/runbooks/downtime_incident_management.md

```diff
@@ -0,0 +1,32 @@
+# Downtime Incident Management Runbook
+
+ Our team has agreed upon steps for handling incidents that cause our site to go offline or become unusable for users. For this document, an incident refers to one in which manage.get.gov is offline or displaying error 400/500 HTTP errors on all pages. However, for this document to apply the cause of the problem must be a critical bug in our code or one of our providers having an outage, not to be confused with a cyber security incident. This document should not be used in response to any type of cyber security incident.
+
+## Response management rules
+
+The following set of rules should be followed while an incident is in progress.
+
+- The person who first notices that the site is down is responsible for using @here and notifying in #dotgov-announce that production is down.
+  - This applies to any team member, including new team members and non-developers.
+- If no engineer has acknowledged the announcement within 10 minutes, whoever discovered the site was down should call each developer via the Slack DM huddle feature. If there is no response, this should escalate to a phone call.
+  - When calling, go down the [phone call list](https://docs.google.com/document/d/1k4r-1MNCfW8EXSXa-tqJQzOvJxQv0ARvHnOjjAH0LII/edit) from top to bottom until someone answers who is available to help.
+  - If this incident occurs outside of regular working hours, choosing to help is on a volunteer basis, and answering a call doesn't mean an individual is truly available to assist.
+- Once an engineer is online, they should immediately start a huddle in the #dotgov-redalert channel to begin troubleshooting.
+- All available engineers should join the huddle once they see it.
+  - If downtime occurs outside of working hours, team members who are off for the day may still be pinged and called but are not required to join if unavailable to do so.
+- Uncomment the banner on get.gov, so it is transparent to users that we know about the issue on manage.get.gov.
+  - Designers or Developers should be able to make this change; if designers are online and can help with this task, that will allow developers to focus on fixing the bug.
+- If the problem is not solved within three hours, change the rules on the [Cloudflare's DNS dashboard](https://dash.cloudflare.com/) so that navigating to manage.get.gov redirects users to get.gov. This will help them see the banner on get.gov informing them that this is a known problem.
```

Comment:
```suggestion
- If the problem is not solved within three hours, change the rules on the [Cloudflare's DNS dashboard](https://dash.cloudflare.com/) so that navigating to manage.get.gov redirects users to get.gov. (See [this document](https://docs.google.com/document/d/1PiWXpjBzbiKsSYqEo9Rkl72HMytMp7zTte9CI-vvwYw/edit) for how to make these changes and who can make them.) This will help them see the banner on get.gov informing them that this is a known problem.
```

Reply:
@abroddrick I propose we test this redirect as configured after-hours sometime before this PR is merged.

Reply:
We have a ticket to investigate if the redirect is the right call and how to best do the redirect. Do we then want to block this ticket by that one? Also do we want to test it on our actual site or with something like dotgov.gov? If we do our site, I think we should show a maintenance banner in advance saying we will be offline for maintenance between X and Y and maybe show a maintence banner on get.gov too. Thoughts?

Reply:
Ah, I [see that now](https://github.com/cisagov/manage.get.gov/issues/1785) and thought we'd already done that given the language about redirects. 

I defer to you. We could drop this language and wait for research to inform our approach (and merge this now) or we could hold this for research and update it more then.

Reply:
Good call, I just left this here and figured it would be updated in that ticket. However, yes I will remove that piece shortly and then we can just add it back in after we confirm that's the right step and have a document with the step by step instructions

Reply:
👍